content/docs/04.5-devx/05-app-profile/08-services/50-service-listings/2-postgresql-db.md

```diff
@@ -0,0 +1,147 @@
+---
+
+import Tabs from 'shared/components/ui/Tabs';
+import WarningBox from 'shared/components/WarningBox';
+import InfoBox from 'shared/components/InfoBox';
+
+
+# Postgres
+
+Palette supports [Postgres](https://www.postgresql.org/) as a database service. Postgres is a powerful open-source object-relational database system with over 35 years of active deployment with a strong reputation for reliability, feature robustness, and performance. Postgres uses and extends the SQL language combined with many features that safely store and scale the most complicated data workloads.
+
+<br />
+
+## Palette Postgres Requirement
+
+The following are requirements for using Palette Postgres:
+
+* Do not use the Postgres user names, `postgres` and `admin`. These user names are reserved for internal system operations and will cause internal conflicts.
+
+
+* The user name format does not support the special character hyphen(-). Example: `name-1` is not supported. 
+
+
+* Clients must set `sslMode=require` or greater as the server instance requires encryption for all connections. Review the [Postgres SSL documentation](https://www.postgresql.org/docs/current/libpq-ssl.html) to learn more about the SSL modes.
+
+
+# Add Postgres to an App Profile
+
+
+## Prerequisite
+
+A Spectro Cloud [account](https://www.spectrocloud.com/get-started/)
+
+## Enablement
+
+You can use the following steps to learn how to add Postgres to your app profile.
+
+1. Log in to [Palette](console.spectrocloud.com).
+
+
+2. On the right side of the window, click the **User Menu** to expand it and select **Switch to App Mode**.
+
+
+3. Navigate to the left **Main Menu** and click on **App Profiles** to create a [new App Profile](/devx/app-profile/create-app-profile/). Provide the following basic information and click **Next**.
+
+|         **Parameter**   | **Description**  |
+|-------------------------|-----------------|
+|Application Profile Name | A custom name for the App Profile.|
+|Version (optional)       | The default value is 1.0.0. You can create multiple versions of an App Profile using the format **`major.minor.patch`**.
+|Description (optional)   | Description of the App Profile. | 
+|Tag (optional)           | Assign tags to the app profile.|
+
+
+4. Select **Postgres** from the database services and start the configuration.
+  
+
+5. Provide the following information to the wizard:
+
+  * **Name:** The database service name. You can use the auto-generated name or create a custom name.
+
+
+  * **Username:** The user name for database access control. 
+
+
+  * **Password:** Security password for the DB service.
+
+<InfoBox>
+
+
+You can use the default system-generated password. If the default password is used, it can be retrieved from the Postgres secret that is created for the password. Review the [Database Password](/devx/app-profile/services/service-listings/postgresql-db/#database-password) section for guidance.
```

Comment:
```suggestion
You can use the default system-generated password. If the default password is used, it can be retrieved from the Postgres Secret that is created for the password. Review the [Database Password](/devx/app-profile/services/service-listings/postgresql-db/#database-password) section for guidance.
```